gallery/src/main.scss

```diff
@@ -61,3 +61,11 @@
 
   position: relative;
 }
+
+.gallery-example-image {
+  max-width: 100%;
+}
+
+.nx-code.nx-code--pre-block {
+  white-space: pre;
```

Comment:
Is this still used?

Reply:
> Is this still used?

Nope: https://github.com/sonatype/sonatype-react-shared-components/pull/28/commits/f592da532d902417ba296aad137ca99885bbf644